### ISSUE TITLE
WasmEdge 0.9.1 states "Image Extensions not supported"

### ISSUE THREAD

**Issue by chairwa** [WasmEdge/WasmEdge#1501]
## Description

I use WasmEdge 0.9.1 and WasmEdge-go 0.9.2 to run
https://github.com/yomorun/yomo-wasmedge-tensorflow

### Current State

curl -sSf https://raw.githubusercontent.com/WasmEdge/WasmEdge/master/utils/install.sh | bash -s -- -e all -v 0.9.1

Detected Linux-aarch64
No root permissions.
Installation path found at /home/firefly/.wasmedge
No path provided
Installing in /home/firefly/.wasmedge
WasmEdge Installation at /home/firefly/.wasmedge
Fetching WasmEdge-0.9.1
/tmp/wasmedge.93909 /tmp
######################################################################## 100.0%
/tmp
Installing WasmEdge-0.9.1-Linux in /home/firefly/.wasmedge/include
Installing WasmEdge-0.9.1-Linux in /home/firefly/.wasmedge/lib
Installing WasmEdge-0.9.1-Linux in /home/firefly/.wasmedge/bin
All extensions
Image Extensions not supported
Tensorflow extensions not supported

source /home/firefly/.wasmedge/env to use wasmedge binaries

### Expected

WasmEdge-image installed successfully.

https://github.com/second-state/WasmEdge-image/releases/tag/0.9.1
Here, aarch-linux binary provided.

## Environment

- Hardware Architecture: aarch64
- Operating system: Ubuntu 20.04

**Comment by SAtacker** [comment]
Please have a look at following cases.
```
satacker@ubuntu:~/wasm_work/WasmEdge$ printf %s\\n%s "0.9.1-beta.1" "0.9.1" | sort --version-sort
0.9.1
0.9.1-beta.1
satacker@ubuntu:~/wasm_work/WasmEdge$ printf %s\\n%s "0.9.1-beta.1" "0.9.2" | sort --version-sort
0.9.1-beta.1
0.9.2
satacker@ubuntu:~/wasm_work/WasmEdge$ printf %s\\n%s "0.9.1-beta.1" "0.9.1" | sort --version-sort
0.9.1
0.9.1-beta.1
satacker@ubuntu:~/wasm_work/WasmEdge$ printf %s\\n%s "0.9.1-beta.1" "0.9.1-rc.1" | sort --version-sort
0.9.1-beta.1
0.9.1-rc.1
satacker@ubuntu:~/wasm_work/WasmEdge$ printf %s\\n%s "0.9.1-rc.1" "0.9.2" | sort --version-sort
0.9.1-rc.1
0.9.2
satacker@ubuntu:~/wasm_work/WasmEdge$ printf %s\\n%s "0.9.1-rc.1" "0.9.1" | sort --version-sort
0.9.1
0.9.1-rc.1
``` 

I will write a tweaked version of sort.
Thanks

**Comment by hydai** [comment]
Hi @chairwa 
This issue should be resolved after #1506 is merged. Please try again. Thanks.
